py3status/modules/backlight.py

```diff
@@ -60,14 +59,27 @@ class Py3status:
     button_up = 4
     cache_timeout = 10
     device = None
-    device_path = None
     format = u'☼: {level}%'
 
+    class Meta:
+        deprecated = {
+            'rename': [
+                {
+                    'param': 'device_path',
+                    'new': 'device',
+                    'msg': 'obsolete parameter use `device`',
+                },
+            ],
+        }
+
     def post_config_hook(self):
         if not self.device:
-            self.device_path = get_device_path()
-        else:
-            self.device_path = "/sys/class/backlight/%s" % self.device
+            self.device = get_device()
+        elif '/' not in self.device:
+            self.device = "/sys/class/backlight/%s" % self.device
+        if self.device is None:
```

Comment:
@tobes Here, we stop `device_path` from being useless as it gets overwritten by either `get_device_path` or `path % self.device`... Bugfix. We go ahead and deprecate anyway. IRC for `nvidia_gpu` issue?